e2e/process-services-cloud/edit-task-filters-component.e2e.ts

```diff
@@ -31,15 +32,28 @@ describe('Edit task filters cloud', () => {
         const tasksCloudDemoPage = new TasksCloudDemoPage();
         const settingsPage = new SettingsPage();
         let tasksService: TasksService;
+        let identityService: IdentityService;
+        let groupIdentityService: GroupIdentityService;
+        let rolesService: RolesService;
+        let testUser, apsUserRoleId, groupInfo;
+        const apiService = new ApiService(browser.params.config.oauth2.clientId, browser.params.config.bpmHost, browser.params.config.oauth2.host, 'BPM');
```

Comment:
The bpm should be taken from the config as well and not hardcoded

Reply:
This will be left as it is for now. Checked and confirmed with Eugenio.